modules/ROOT/pages/_partials/app-security/using-the-admin-ui.adoc

```diff
@@ -27,7 +29,7 @@ This view provides statistics and information about each deployed version of an
 * *Disable on Startup*: Whether this application version is disabled on startup.
 * *Custom Disable Message*: A custom message that is displayed when this version of the mobile app is disabled.
 
-== Enabling and Disabling an application version
+== Enabling and disabling a mobile app
```

Comment:
I think it's important to note that there may be multiple versions of a mobile app, and the MSS allows you to enable/ disable these individually. In that regard I would perhaps suggest:

```suggestion
Enabling and disabling mobile app versions
```